menas/src/main/scala/za/co/absa/enceladus/menas/repositories/OozieRepository.scala

```diff
@@ -363,3 +361,10 @@ class OozieRepository @Autowired() (oozieClientRes: Either[OozieConfigurationExc
     this.writeScheduleData(coordPath, content)
   }
 }
+
+object OozieRepository {
```

Comment:
The companion objects in the other `Repository` classes are at the top of the file. Please follow convention.